Events/src/main/java/com/github/swcraftlyon/meetup/infra/Main.java

```diff
@@ -0,0 +1,26 @@
+package com.github.swcraftlyon.meetup.infra;
+
+import com.github.swcraftlyon.meetup.domain.IPlanner;
+import com.github.swcraftlyon.meetup.domain.entities.Event;
+import com.github.swcraftlyon.meetup.domain.services.EventManager;
+import com.github.swcraftlyon.meetup.domain.services.IEventManager;
+import com.github.swcraftlyon.meetup.infra.adapters.repositories.PlannerRepository;
+import com.github.swcraftlyon.meetup.infra.dao.IEventDAO;
+import com.github.swcraftlyon.meetup.infra.dao.MangoEventDao;
+
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
In our case, we haven't any framework yet? I thought I understood that we don't want to use any framework. at least libraries maybe!?